index.html

```diff
@@ -1,17 +1,137 @@
+                                        class="dropdown-content-span">Reminder</span></a>
+                                <a href="#"><img src="/images/icon-planning.svg"><span
+                                        class="dropdown-content-span">Planning</span></a>
+                            </div>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="dropdown">
+                            <button class="dropbtn">Company
+                                <i class="fa fa-caret-down"></i>
+                            </button>
+                            <div class="dropdown-content">
+                                <a href="#">History</a>
+                                <a href="#">Our Team</a>
+                                <a href="#">Blog</a>
+                            </div>
+                        </div>
+                    </li>
+
+                    <li>
+                        <div class="test"><a href="#">Career</a></div>
+                    </li>
+                    <li>
+                        <div class="test">
+                            <a href="#">About</a>
+                        </div>
+                    </li>
+                </ul>
+            </nav>
+        </div>
+        <div id="rightLinks">
+            <nav>
+                <ul>
+                    <li id="login"><a href="#">Login</a></li>
+                    <li id="signup"><a href="#">Register</a></li>
+                </ul>
+            </nav>
+        </div>
+        <!-- header for mobile version starting -->
+        <div id="hamburger-icon" onclick="toggleMobileMenu(this)">
+            <div class="bar1"></div>
+            <div class="bar2"></div>
+            <div class="bar3"></div>
+            <ul class="mobile-menu">
+                <li><a href="/home">Home</a></li>
+                <li><a href="/products">Products</a></li>
+                <li><a href="/about">About</a></li>
+                <li id="login"><a href="/login">Login</a></li>
+                <li id="signup"><a href="/signup">Register</a></li>
+            </ul>
+        </div>
+    </header>
+    <!-- main content area is starting from here -->
+    <div class="mainContainer">
+
+        <div class="leftColumn">
+            <div class="mainHeading">
+                <h1>Make</h1>
+                <h1>remote work</h1>
+            </div>
+            <div class="mainText">
+                <p>
+                    Get your team in sync, no matter your location. Streamline processes,
+                    create team rituals, and watch productivity soar.
+
+
+                </p>
+            </div>
+
+            <div class="mainButton">
+                <a href="#">Learn more</a>
+            </div>
+
+            <div class="bottomStyle">
+                <ul>
+                    <li>
+                        <img src="/images/client-databiz.svg">
```

Comment:
Non decorative images must have `alt` attribute explaining what the image is. This way users relying on screen readers can still use the page.